main/src/ca/uwaterloo/flix/language/phase/CreateExecutableAst.scala

```diff
@@ -197,15 +197,15 @@ object CreateExecutableAst {
         val fvs = freeVars.map(CreateExecutableAst.toExecutable).toArray
         ExecutableAst.Expression.MkClosureRef(e.asInstanceOf[ExecutableAst.Expression.Ref], fvs, tpe, loc)
       case SimplifiedAst.Expression.ApplyRef(name, args, tpe, loc) =>
-        val argsArray = args.map(toExecutable).toArray
+        val argsArray = args.map(toExecutable)
```

Comment:
If the arguments are no longer an array, then you should probably update the variable names. Or just inline the call, e.g.:

```scala
ExecutableAst.Expression.ApplyRef(name, args.map(toExecutable), tpe, loc)
```